make/migrations/postgresql/0090_2.6.0_schema.up.sql

```diff
@@ -1,2 +1,18 @@
 /* Correct project_metadata.public value, should only be true or false, other invaild value will be rewrite to false */
-UPDATE project_metadata SET value='false' WHERE name='public' AND value NOT IN('true', 'false');
\ No newline at end of file
+UPDATE project_metadata SET value='false' WHERE name='public' AND value NOT IN('true', 'false');
+
+/*
+System Artifact Manager
+Github proposal link : https://github.com/goharbor/community/pull/181
+*/
+ CREATE TABLE IF NOT EXISTS "system_artifact" (
```

Comment:
to keep code style consistent, could you use the raw attributie without double quotation marks？

Reply:
Done